src/main/scheduler/scheduler.h

```diff
@@ -153,7 +160,7 @@ typedef struct {
     bool (*checkFunc)(timeUs_t currentTimeUs, timeDelta_t currentDeltaTimeUs);
     void (*taskFunc)(timeUs_t currentTimeUs);
     timeDelta_t desiredPeriod;      // target period of execution
-    const uint8_t staticPriority;   // dynamicPriority grows in steps of this size, shouldn't be zero
+    const int8_t staticPriority;   // dynamicPriority grows in steps of this size, shouldn't be zero
```

Comment:
Is this comment still true?

Reply:
Yes, this is still true. For all the non-realtime tasks the dynamic priority is "aged" by the static priority * the waiting loops. This way the higher static priority tasks end at the head of the queue.

Reply:
Sorry for not being specific, the question was about the 'shouldn't be zero', can't this be set to `TASK_PRIORITY_IDLE`?

Reply:
Ah, yes you're correct. The "shouldn't be zero" part is not correct. I'm assuming that the `TASK_PRIORITY_IDLE` was added after the initial implementation and this got missed. Nothing has changed in regards to this in this PR. But I'll update the comment.

`TASK_PRIORITY_IDLE` does have the possibility of starving those tasks if processing load was high. This should probably be improved in the future (again, no change in this PR). Of the tasks tagged IDLE `TASK_PINIOBOX` is the one I'd be most concerned about. Followed a lesser extent by `TASK_VTXCTRL` and `TASK_CAMCTRL`. We may want to consider bumping `TASK_PINIOBOX` to `TASK_PRIORITY_LOW` in another PR.